### PR TITLE
Test WalImageFile

### PR DESCRIPTION
**Pull request by hugovk**
WAL is a proprietary 2D image format for Quake 2.

hopper.wal was created by opening hopper.png in [Wally](http://www.cs.rochester.edu/~brown/242/assts/mazedoc.html) and exporting it.

Helps towards #722.

### CODE REVIEW

**Comment by hugovk** [comment]
Like many of these old, untested formats...

`im.show()` gives:

image

`self.assert_image_similar(im, hopper("P", epsilon)` needs an epsilon of ~86.5.

Re-opening hopper.wal in Wally gives:

image

File: Tests/test_file_wal.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    def test_open(self):
+        # Arrange
+        TEST_FILE = "Tests/images/hopper.wal"
+
+        # Act
+        im = WalImageFile.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "WAL")
+        self.assertEqual(im.format_description, "Quake2 Texture")
```
Comment (radarhere) [inline Tests/test_file_wal.py]: Just a question - why assert the `format_description`? Once the `format` is asserted, this is a given. `format_description` isn't tested anywhere else.

**Comment by wiredfool** [comment]
Is it a paletted image?  Or perhaps a low bit depth image returning (e.g.) 0-31 that needs to be stretched to 0-255 to match brightness?

**Comment by hugovk** [comment]
Merging this to include the basic tests. Further issues/PRs welcome from interested parties to fix the loading.

**Pull request merged**